src/control/lib/hardware/fabric.go

```diff
@@ -970,7 +977,9 @@ func (s *FabricScanner) Scan(ctx context.Context, providers ...string) (*FabricI
 		if len(providers) == 0 {
 			return nil, errors.New("no fabric interfaces found")
 		}
-		return nil, fmt.Errorf("no fabric interfaces found with providers: %s", strings.Join(providers, ", "))
+		return nil, fmt.Errorf("no fabric interfaces found with %s %q",
+			common.Pluralise("provider", len(providers)),
+			strings.Join(providers, ", "))
```

Comment:
I thought this was better than `no fabric interfaces found with providers: ` that I was seeing before the change but I will remove. Done.

Reply:
It definitely does look better for a single provider, but it is just odd for multiples, where I'd expect something like `"ofi+tcp", "ofi+verbs"` instead of lumping them all in one quoted string. Up to you whether you agree or disagree, I wouldn't block on it.

Reply:
done